include/private/config/locale/bg-bg.h

```diff
@@ -280,4 +284,3 @@ L"Stumpless registration of Windows Event Log Source"
 #  define L10N_WSAGETLASTERROR_ERROR_CODE_TYPE \
 "резултатът от WSAGetLastError след неуспешното извикване"
 
-#endif /* __STUMPLESS_PRIVATE_CONFIG_LOCALE_BG_BG_H */
```

Comment:
I'm not sure why the ending `#endif` statements have been dropped here. I just tested the add_l10n_string script and it does not seem to do this - how did you modify these files?

Reply:
I did it through the script. Don't know why it dropped them ill check again.

Reply:
If you are able to reproduce this with the script, please share the version of Ruby and command invocation used and I will see if I can find it.